include/solve_lp.h

```diff
@@ -25,7 +25,9 @@
 #include <stdio.h>
 #include <cmath>
 #include <exception>
-#include "run_headers/lp_lib.h"
+#undef Realloc
```

Comment:
why is this needed?

Reply:
#undef Realloc
#undef Free

Are needed fro the conflicts between R.h and windows.h when we install the package at windows.